sim/default.py

```diff
@@ -14,18 +14,21 @@
 back_camera_resolution = rospy.get_param('sim/back_camera_resolution', None)
 bottom_camera_resolution = rospy.get_param('sim/bottom_camera_resolution', None)
 create_teleport_actuator = rospy.get_param('sim/create_teleport_actuator', False)
+prototype_uav = rospy.get_param('sim/prototype_uav', True)
+num_roombas = rospy.get_param('sim/num_roombas',True)
+num_obstacle_roombas = rospy.get_param('sim/num_obstacles',True)
 
-num_targets = rospy.get_param('sim/num_roombas')
-num_obstacles = rospy.get_param('sim/num_obstacles')
 
+num_targets = num_roombas
+num_obstacles = num_obstacle_roombas
```

Comment:
Was there a reason to change the way the variables `num_targets` and `num_obstacles` are initialized?

Reply:
I did this so we aren't getting the sim/num_roombas parameter twice

Reply:
Why can't we just set `num_targets` and `num_obstacles` like they were before?  I think we both added the same feature and the merge messed it up

Reply:
fixed in 11056f1